app/controllers/application_controller/ci_processing.rb

```diff
@@ -892,6 +892,7 @@ def reconfigure
     @force_no_grid_xml   = true
     @view, @pages = get_view(Vm, :view_suffix => "VmReconfigureRequest", :where_clause => ["vms.id IN (?)", @reconfigure_items])  # Get the records (into a view) and the paginator
     get_reconfig_limits
+    @supports_reconfigure_disks = supports_reconfigure_disks?
```

Comment:
Dont know much about the UI, but cant we do this without a instance variable? Why no call the method directly?